### ISSUE TITLE
Store result as JSON object?

### ISSUE THREAD

**Issue by depsimon** [YieldStudio/nova-google-autocomplete#1]
Hi team, 

Could you make it so it's possible to retrieve the address object (entirely or partially) as a JSON object?

In my case I want to store anything related to the place as JSON.

I was wondering if it was possible to make it easy to store the value as a JSON object instead as a string with the address?

I'd like to store all values returned by the API so that I can use them later or in other fields.

I currently do it like this:

```php
GoogleAutocomplete::make('Geolocation')->countries('BE')->withValues(['latitude', 'longitude', 'street_number', 'route', 'locality', 'administrative_area_level_1', 'country', 'postal_code', 'adr_address', 'aspects', 'business_status', 'formatted_phone_number', 'html_attribution', 'icon', 'icon_background_color', 'icon_mask_base_uri', 'international_phone_number', 'name', 'opening_hours', 'photos', 'place_id', 'plus_code', 'price_level', 'rating', 'reviews', 'types', 'url', 'user_ratings_total', 'utc_offset_minutes', 'vicinity', 'website']),

AddressMetadata::make('Coordinates', 'coordinates')->fromValue('{{latitude}}, {{longitude}}'),
AddressMetadata::make('Address', 'address')->fromValue('{
    "latitude": "{{latitude}}",
    "longitude": "{{longitude}}",
    "street_number": "{{street_number}}",
    "route": "{{route}}",
    "locality": "{{locality}}",
    "administrative_area_level_1": "{{administrative_area_level_1}}",
    "country": "{{country}}",
    "postal_code": "{{postal_code}}",
    "adr_address": "{{adr_address}}",
    "aspects": "{{aspects}}",
    "business_status": "{{business_status}}",
    "formatted_phone_number": "{{formatted_phone_number}}",
    "html_attribution": "{{html_attribution}}",
    "icon": "{{icon}}",
    "icon_background_color": "{{icon_background_color}}",
    "icon_mask_base_uri": "{{icon_mask_base_uri}}",
    "international_phone_number": "{{international_phone_number}}",
    "name": "{{name}}",
    "opening_hours": "{{opening_hours}}",
    "photos": "{{photos}}",
    "place_id": "{{place_id}}",
    "plus_code": "{{plus_code}}",
    "price_level": "{{price_level}}",
    "rating": "{{rating}}",
    "reviews": "{{reviews}}",
    "types": "{{types}}",
    "url": "{{url}}",
    "user_ratings_total": "{{user_ratings_total}}",
    "utc_offset_minutes": "{{utc_offset_minutes}}",
    "vicinity": "{{vicinity}}",
    "website": "{{website}}"
}'),
```

But it doesn't parse objects (such as photos) correctly:

```json
"photos": "[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]",
```

**Comment by JamesHemery** [comment]
Hi @depsimon,

It is possible and would be an interesting use case. It should not require many modifications but we do not have time to look into it at the moment.

We are open to review/merge a PR.

Regarding the method you tried, it should work, but the first braces (of the json) should conflict when parsing.

Another solution would be to add a {{response}} key that injects the json.

**Comment by JamesHemery** [comment]
Released as 1.1.0 :) Thank you!
